cpp/core/compute/Backend.h

```diff
@@ -73,14 +74,19 @@ class Backend : public std::enable_shared_from_this<Backend> {
     return substraitPlan_;
   }
 
+  virtual MemoryManager*
+  getMemoryManager(std::string name, std::shared_ptr<MemoryAllocator> allocator, std::shared_ptr<AllocationListener>) {
+    throw GlutenException("Not implement getMemoryManager");
```

Comment:
Can we correct the wordings here, and those in other member functions in this file?

e.g. "Not implement getMemoryManager" -> "getMemoryManager not implemented"